### PR TITLE
fix importCSVTool import directory bug & encode bug

### PR DESCRIPTION
**Pull request by mychaow**
1. When import files from a directory, the previous code will close session after import one file. So, the next file import will be failed.
2. The previous code failed to import Chinese character.

### Content3 ...

<hr>

This PR has:
- [ ] been self-reviewed.
    - [ ] concurrent read
    - [ ] concurrent write
    - [ ] concurrent read and write 
- [ ] added documentation for new or modified features or behaviors.
- [ ] added Javadocs for most classes and all non-trivial methods. 
- [ ] added or updated version, __license__, or notice information
- [ ] added comments explaining the "why" and the intent of the code wherever would not be obvious 
  for an unfamiliar reader.
- [ ] added unit tests or modified existing tests to cover new code paths, ensuring the threshold 
  for code coverage.
- [ ] added integration tests.
- [ ] been tested in a test IoTDB cluster.

<hr>

### CODE REVIEW

**Comment by HTHou** [comment]
Hi @mychaow, can you please add some bug description?

File: cli/src/main/java/org/apache/iotdb/tool/ImportCsv.java [hunk 1]
```diff
@@ -169,14 +172,6 @@ private static void loadDataFromCSV(File file) {
...
     } catch (IoTDBConnectionException | StatementExecutionException e) {
       System.out.println("Meet error when insert csv because " + e.getMessage());
-    } finally {
-      try {
-        if (session != null) {
-          session.close();
-        }
-      } catch (IoTDBConnectionException e) {
-        System.out.println("Sql statement can not be closed because: " + e.getMessage());
-      }
```
Comment (HTHou) [inline cli/src/main/java/org/apache/iotdb/tool/ImportCsv.java:179]: Hi, if remove these lines, how to close the session?

Comment (mychaow) [reply cli/src/main/java/org/apache/iotdb/tool/ImportCsv.java:179]: session will be closed in the importCsvFromFile method which calls this method. So,  closing session is unnecessary here.

**Review by HTHou — APPROVED** [review]
LGTM

**Pull request merged**